torch/onnx/utils.py

```diff
@@ -1161,12 +1170,21 @@ def _run_symbolic_function(g, block, n, inputs, env, operator_export_type=Operat
             attrs = {k + "_" + n.kindOf(k)[0]: n[k] for k in n.attributeNames()}
             outputs = n.outputsSize()
             attrs["outputs"] = outputs
-            return g.at(op_name, *inputs, **attrs)
+            # `overload_name` is set for non-Caffe2 builds only
+            return g.at(op_name, *inputs, overload_name=_get_aten_op_overload_name(n), **attrs)
         else:
             raise sym_registry.UnsupportedOperatorError(domain, op_name, opset_version)
     except RuntimeError:
+        # TODO: Should RuntimeError be used for unsupported operators?
```

Comment:
I think a custom exception type makes sense but I would file an issue in GitHub rather than adding a TODO comment here.